### DISCUSSION TITLE
No grid information available

### DISCUSSION THREAD

**Discussion by jamos104** [tijsverkoyen/HomeAssistant-FusionSolar#204] [Q&A]
Hi, as per the instrructions below I am trying to set up ny energy tab. I can get the Solar Panen info, but none of the grid info. Not sure if this is a common issue in Australian solar configurations. I've allowed all data in the Northbound.

The first thing to configure is the electricity grid:

Grid consumption is given by sensor.xxx_reverse_active_energy
Return to grid is given by sensor.xxx_active_energy_forward_active_energy
For solar panels, the sensor is sensor.xxx_total_current_day_energy

**Comment by tijsverkoyen** [comment]
I suspect you have only access to the inverter but not any power meters.
It depends on your setup and which devices are used.

**Reply by jamos104** [reply]
I thought the inverter would tell me whats being exported to the grid. I'll contact their tech support
